src/cmap/commands.ts

```diff
@@ -84,8 +84,8 @@ export class Query {
 
     // Validate that we are not passing 0x00 in the collection name
     if (ns.indexOf('\x00') !== -1) {
-      // TODO(NODE-3483): Replace with MongoCommandError
-      throw new MongoDriverError('Namespace cannot contain a null character');
+      // TODO(NODE-3483): Use MongoNamespace static method
+      throw new MongoDriverError('namespace cannot contain a null character');
```

Comment:
Why the change to start with a common letter here?

Reply:
Oops, this got messed up in the rebase.